script/send_new_webmentions

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env ruby
+
+ENV["RAILS_ENV"] ||= "production"
+
+APP_PATH = File.expand_path('../../config/application', __FILE__)
+require File.expand_path('../../config/boot', __FILE__)
+require APP_PATH
+Rails.application.require_environment!
+
+LAST_STORY_KEY = "webmentions:last_story_id".freeze
+
+def endpoint_from_body(html)
+  doc = Nokogiri::HTML(html)
+
+  if !doc.css('[rel~="webmention"]').css('[href]').empty?
+    doc.css('[rel~="webmention"]').css('[href]').attribute('href').value
+  elsif !doc.css('[rel="http://webmention.org/"]').css('[href]').empty?
+    doc.css('[rel="http://webmention.org/"]').css('[href]').attribute('href').value
+  elsif !doc.css('[rel="http://webmention.org"]').css('[href]').empty?
+    doc.css('[rel="http://webmention.org"]').css('[href]').attribute('href').value
+  end
+end
+
+def endpoint_from_headers(header)
+  return unless header
+
+  if (matches = header.match(/<([^>]+)>; rel="[^"]*\s?webmention\s?[^"]*"/))
```

Comment:
Similarly, it's gotta be possible to make this two regexps. This could use some tests so there's samples of valid data, I'm puzzled by the shape of this being "url; webmation" *and* "webmention; url", that's a surprisingly flexible grammar.